tuf/signed/interface.go

```diff
@@ -47,7 +39,6 @@ type KeyService interface {
 // CryptoService defines a unified Signing and Key Service as this
```

Comment:
Yeah, good call. Would definitely make sense to just replace CryptoService with KeyService. That touches a lot of things so I'll do that in a separate PR.